website/content/docs/job-specification/consul.mdx

```diff
@@ -231,6 +230,56 @@ job "docs" {
 }
 ```
 
+### Consul Admin Partition
+
+This example demonstrates how to configure Consul admin partitions for different
+tasks within a group. The Consul client agent must separately specify the admin
+partition in the agent configuration. Refer to the Consul documentation's
+[agent configuration reference][] for more information.
+
+<EnterpriseAlert />
+
+In the following example, the `template` block in the `web` and `app` tasks use
+the default Consul cluster. It obtains a token that allows it access to the `prod`
+admin partition in Consul.
```

Comment:
This is kind of a weird example to have without further explanation, because you can't have two `task`-level Consul blocks in different admin partitions (because tasks are placed together in the same Allocation). I'd probably move the example `consul` block up to the `group` level and then have a note here that if you have `task`-level Consul blocks that they need to have the same partition.